chunky/src/java/se/llbit/chunky/renderer/scene/Scene.java

```diff
@@ -358,8 +357,8 @@ public static void exportToZip(String name, File targetFile) {
   public synchronized void initBuffers() {
     frontBuffer = new BitmapImage(width, height);
     backBuffer = new BitmapImage(width, height);
-    alphaChannel = new byte[width * height];
-    samples = new double[width * height * 3];
+    samples = new SampleBuffer(width,height);
+    samples.alphaInit();
```

Comment:
If alpha is _always_ enabled anyway, it could just be initialized in the constructor. If not, it could only be initialized if the sky is set to transparent to improve memory usage.